.github/workflows/mirror_repos.yml

```diff
@@ -45,22 +45,22 @@ jobs:
           token: ${{ secrets.AZTEC_BOT_GITHUB_TOKEN }}
       - name: Push to aztec-nr repo
         run: |
-          SUBREPO_PATH=yarn-project/aztec-nr
+          SUBREPO_PATH=aztec-nr
           git config --global user.name AztecBot
           git config --global user.email tech@aztecprotocol.com
 
           monorepo_url="$GITHUB_SERVER_URL/$GITHUB_REPOSITORY"
           # list all aztec-packages tags, take the "highest" version
           monorepo_tag="$(git tag --list aztec-packages-v* | sort --version-sort | tail -1)"
-          monorepo_protocol_circuits_path="yarn-project/noir-protocol-circuits"
+          monorepo_protocol_circuits_path="noir-protocol-circuits"
```

Comment:
I'm a little surprised to find this top-level, but I guess we consider this one of our major components?